ACPs/77-reinventing-subnets/README.md

```diff
@@ -1,24 +1,24 @@
-| ACP | 77 |
-| :--- | :--- |
-| **Title** | Reinventing Subnets |
-| **Author(s)** | Dhruba Basu ([@dhrubabasu](https://github.com/dhrubabasu)) |
-| **Status** | Proposed ([Discussion](https://github.com/avalanche-foundation/ACPs/discussions/78)) |
-| **Track** | Standards |
-| **Replaces** | [ACP-13](../13-subnet-only-validators/README.md) |
+| ACP           | 77                                                                                   |
```

Comment:
This alignment actually isn't necessary in the markdown tables, so I think leaving it as is makes it easier to update in the future if there any text changes, etc.

Reply:
my linter actually auto corrects this every time i save 😅 but i can prob disable it